src/main/scala/mesosphere/marathon/core/event/Events.scala

```diff
@@ -9,9 +10,20 @@ import mesosphere.marathon.core.instance.Instance
 import mesosphere.marathon.state.{ AppDefinition, PathId, Timestamp }
 import mesosphere.marathon.upgrade.{ DeploymentPlan, DeploymentStep }
 import org.apache.mesos.{ Protos => Mesos }
+import play.api.libs.json.Json
 
 import scala.collection.immutable.Seq
 
+trait Event {
+  val eventType: String
+  val message: String
+}
+
+case class JsonEvent(event: MarathonEvent) extends Event {
+  val eventType: String = event.eventType
+  lazy val message: String = Json.stringify(eventToJson(event))
+}
+
 sealed trait MarathonEvent {
   val eventType: String
   val timestamp: String
```

Comment:
sorry for forgetting about this... since we control the serialization/deserialization of these, we could have the jsonString as a member and not serialize the field.